public/js/filter_spreadsheet_table.js

```diff
@@ -0,0 +1,11 @@
+jQuery(function ($) {
+  'use strict';
+
+  	// Filter Spreadsheet table by entered value in search field
+	$('.spreadsheet-table-search').keyup(function() {
```

Comment:
Please use a different event than `keyup`. Keyup triggers at literally any keyup, also when the values don't change (like arrow keys, tabbing to the field, ...). Keyup also doesn't trigger when pasting using the mouse. Probably the `change` event would be better here.